public/views/index.html

```diff
@@ -274,8 +274,16 @@
         var preloader = document.getElementsByClassName("preloader");
         if (preloader.length) {
           preloader[0].className = "preloader preloader--done";
-          // NI fork: Tells iframe-host loading indicator when to disappear
-          window.parent.postMessage({ type: 'iframeLoaded' }, window.parent.location.origin);
+
+          // NI fork: Ensure that grafana is loaded within an iframe unless it is running locally
```

Comment:
If I'm following we have this script twice, once in the app load script (assuming the app loads successfully) and once in the grafana failed load case (in case the app fails to load).

This happens much later in the page life-cycle after downloading and initializing a bunch of stuff. Is it possible to create a new `<script>` section at the end of the `<head>` or the start of the `<body>` that does this stuff so that we can do the navigation ASAP?

Reply:
Actually if we did it between the  `<base>` and the first `<link>` tag that would be the earliest possible. It would avoid waiting for the CSS files to load as well.